<?xml version="1.0"?>

<launch>
<param name="use_sim_time" value="false"/>
<arg name="min_altitude" default="0.5"/>
    <node name="audio_feedback_node" pkg="training_q_learning" type="audio_feedback.py" output="screen" >
        <param name = "min_altitude" value = "$(arg min_altitude)" />
        <param name = "mp_edge_length" value = "$(arg mp_edge_length)" />
    </node> 
</launch>
